pkg/doc/verifiable/test-suite/verifiable_suite_test.go

```diff
@@ -149,7 +150,7 @@ func parseKeys(packedKeys string) (private, public interface{}) {
 
 	rs256PrivateKeyJwk, exist := decodedJwtMap["rs256PrivateKeyJwk"]
 	if !exist {
-		log.Println(fmt.Errorf("cannot get rs256PrivateKeyJwk key"))
+		log.Println(errors.New("cannot get rs256PrivateKeyJwk key"))
```

Comment:
All logging should use the framework logger.

Reply:
Got it. Changed.